### DISCUSSION TITLE
Required a WebSocket RPC for Polygon-Edge Chain

### DISCUSSION THREAD

**Discussion by Noooblien** [0xPolygon/polygon-edge#509] [Q&A] [answered]
### Description
I am making a block explorer of a chain that I have made using Polygon Edge and I require a WebSocket RPC URL for Block Explorer so it can auto reload the information 
I am not able to figure it out how I can get a WebSocket

**Comment by lazartravica** [comment, marked as answer]
Hello @rahul-spirit 

Here are the basic steps for instantiating a WS connection:

You can use this tool for testing WS quickly:
https://github.com/vi/websocat

By default when you run the polygon SDK, it sets up the WS interface, so there is no need to additionally configure it

Opening a WS connection with websocat tool:

```
websocat ws://127.0.0.1:8545/ws
```

:point_up_2:

This is for a localhost where the HTTP port is 8545. Notice the /ws at the end.

After you’ve opened up the WS connection, you can query it like the regular JSON RPC, by sending a json request to the open connection, for example:

```
{“id”:1, “jsonrpc”: “2.0”, “method”:“eth_blockNumber”,“params”:[]}
```

Let me know how that goes and whether you need any further assistance.
